ui/src/features/Editor/hooks.ts

```diff
@@ -31,18 +40,20 @@ import useUIState from "./useUIState";
 export default ({
   yDoc,
   yWorkflows,
+  yAwareness,
   undoManager,
   undoTrackerActionWrapper,
 }: {
   yDoc: Doc | null;
   yWorkflows: YMap<YWorkflow>;
+  yAwareness?: Awareness | null;
```

Comment:
Also, yAwareness I believe is always going to be there, so you can make it necessary here, and then simplify the yUsers (rawUsers) to just be `const rawUsers = useUsers(yAwareness)`